radixdlt/src/main/java/com/radixdlt/consensus/BFTEventReducer.java

```diff
@@ -201,9 +202,7 @@ public void processProposal(Proposal proposal) {
 			vertexStore.insertVertex(proposedVertex);
 		} catch (VertexInsertionException e) {
 			counters.increment(CounterType.CONSENSUS_REJECTED);
-
-			log.info(String.format("%s: PROPOSAL: Rejected", this.getShortName()), e);
-
+			log.trace("{} PROPOSAL: Rejected. Reason: {}", this::getShortName, e::getMessage);
```

Comment:
I think this should be logged at INFO level at least.  This should be a quite unusual event in the final system, and indicates that some attention is really required.

Reply:
Probably should be a warning.

Reply:
I agree